models/db.py

```diff
@@ -213,7 +213,7 @@ def get_course_url(*args):
 
 
 def getCourseNameFromId(courseid):
-    """ used to compute auth.user.course_name field """
```

Comment:
New version of black?

Reply:
Yes, Python 3.9.8 breaks the older version of Black and causes CI to fail.